deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -126,7 +166,10 @@ export class NetworkStack extends Stack {
                         cidrMask: 24,
                     },
                 ],
+                natGateways: 0,
```

Comment:
What subnets get created with this? Previously was 2 public and 2 private. Would this now be only 2 isolated subnets?

Reply:
I also expected this to be more of a breaking change but the Jenkins job passed...

Reply:
We still have 2 private and 2 public. The public are attached to an Internet Gateway that allows outside communication. The private do not have a NAT gateway so they are isolated within the VPC aside from the VPC Endpoints that connect into it

Reply:
Ah I see